plugin/ochttp/client_test.go

```diff
@@ -119,3 +120,115 @@ func TestClient(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkTransportNoInstrumentation(b *testing.B) {
+	benchmarkClientServer(b, &ochttp.Transport{NoStats: true, NoTrace: true})
+}
+
+func BenchmarkTransportTraceOnly(b *testing.B) {
+	benchmarkClientServer(b, &ochttp.Transport{NoStats: true})
+}
+
+func BenchmarkTransportStatsOnly(b *testing.B) {
+	benchmarkClientServer(b, &ochttp.Transport{NoTrace: true})
+}
+
+func BenchmarkTransportAllInstrumentation(b *testing.B) {
+	benchmarkClientServer(b, &ochttp.Transport{})
+}
+
+// Copied from net/http/serve_test.go
+func benchmarkClientServer(b *testing.B, transport http.RoundTripper) {
+	b.ReportAllocs()
+	b.StopTimer()
+	ts := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(rw, "Hello world.\n")
+	}))
+	defer ts.Close()
+	var client http.Client
+	client.Transport = transport
+	b.StartTimer()
+
+	for i := 0; i < b.N; i++ {
+		res, err := client.Get(ts.URL)
+		if err != nil {
+			b.Fatalf("Get: %v", err)
+		}
+		all, err := ioutil.ReadAll(res.Body)
+		res.Body.Close()
+		if err != nil {
+			b.Fatal("ReadAll:", err)
+		}
+		body := string(all)
+		if body != "Hello world.\n" {
+			b.Fatal("Got body:", body)
+		}
+	}
+
+	b.StopTimer()
```

Comment:
Unnecessary

Reply:
Ugh, don't know where my previous reply went but just to clarify again this is copied from the net/http test so I want to leave it as-is